src/bin/build.rs

```diff
@@ -28,21 +25,16 @@ pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
 // Build command implementation
 pub fn execute(args: &ArgMatches) -> Result<()> {
     let book_dir = get_book_dir(args);
-    let book = MDBook::new(&book_dir).read_config()?;
+    let mut book = MDBook::new(&book_dir).read_config()?;
 
-    let mut book = match args.value_of("dest-dir") {
-        Some(dest_dir) => book.with_destination(dest_dir),
-        None => book,
-    };
+    if let Some(dest_dir) = args.value_of("dest-dir") {
```

Comment:
.. did you move the `destination` because this argument must be independent of the backend?

Reply:
Yeah. I was thinking a renderer would be told to put its stuff in the `$dest-dir/renderer_name/` directory, or `$dest-dir/` if there's only one renderer.

That way renderers are "sandboxed" and should only be touching stuff in the directory they're told to write output to.

Reply:
kk